lib/python/qmk/cli/hello.py

```diff
@@ -3,6 +3,7 @@
 This is an example QMK CLI script.
 """
 from milc import cli
+from qmk.questions import question
```

Comment:
I guess you wanted to do something with this.

Reply:
Good catch, yeah. I was using that command to test.